include/lazy_importer.hpp

```diff
@@ -415,7 +413,7 @@ namespace li { namespace detail {
 
         LAZY_IMPORTER_FORCEINLINE const char* name(size_type index) const noexcept
         {
-            return reinterpret_cast<const char*>(
+            return /*reinterpret_cast<const char*>*/(
```

Comment:
Feel free to remove the comment and parens

Reply:
I forgot to remove the comment before pushing.